pyensign/ensign.py

```diff
@@ -114,21 +112,15 @@ async def publish(
             client_id = str(ULID())
 
         # Ensure the topic ID exists by getting or creating it
-        topic_id = None
-        try:
-            id_str = await self.topic_id(topic_name)
-            topic_id = ULID.from_str(id_str)
-        except EnsignTopicNotFoundError:
-            topic = await self.create_topic(topic_name)
-            topic_id = ULID.from_bytes(topic.id)
+        topic_id = await self.ensure_topic_exists(topic_name)
```

Comment:
I forgot that on publish pyensign was already creating the topic if it doesn't exist, so the user doesn't actually need to use `ensure_topic_exists` if they don't want to, but it did happen to clean up the code here a bit.

Reply:
the oneliner here is 🔥 ;)